.github/workflows/sonarcloud.yml

```
@@ -40,9 +40,15 @@ permissions:
 
 jobs:
   Analysis:
+    if: secrets.SONAR_TOKEN != ''
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)